quolance-api/src/main/java/com/quolance/quolance_api/controllers/ClientController.java

```diff
@@ -55,6 +56,19 @@ public ResponseEntity<String> deleteProject(@PathVariable(name = "projectId") Lo
         return ResponseEntity.ok("Project deleted successfully");
     }
 
+    @PutMapping("/projects/{projectId}")
```

Comment:
no need to pass the id the dto payload can hold it --> "/projects/update"

Reply:
C'est mieux comme ca je trouve